src/App.js

```diff
@@ -44,25 +45,23 @@ class App extends Component {
      */
 
     const contract = require('truffle-contract')
-    const simpleStorage = contract(SimpleStorageContract)
-    simpleStorage.setProvider(this.state.web3.currentProvider)
+    const TrustPool = contract(TrustPoolJson)
+    TrustPool.setProvider(this.state.web3.currentProvider)
 
-    // Declaring this for later so we can chain functions on SimpleStorage.
-    var simpleStorageInstance
+    // Declaring this in the closure for later access
+    window.contractInstance = null
 
     // Get accounts.
     this.state.web3.eth.getAccounts((error, accounts) => {
-      simpleStorage.deployed().then((instance) => {
-        simpleStorageInstance = instance
-
-        // Stores a given value, 5 by default.
-        return simpleStorageInstance.set(5, {from: accounts[0]})
-      }).then((result) => {
-        // Get the value from the contract to prove it worked.
-        return simpleStorageInstance.get.call(accounts[0])
+      TrustPool.deployed().then((instance) => {
+        window.contractInstance = instance
```

Comment:
Maybe in the future, we'll want to put the contract somewhere else, but putting it on the window object is cool for now.